SPECS/tcpdump/tcpdump.spec

```diff
@@ -17,15 +18,15 @@ It allows the user to display TCP/IP and other packets being
 transmitted or received over a network to which the computer is attached.
 
 %prep
-%setup -q
+%autosetup -p1
 
 %build
 %configure
 make %{?_smp_mflags}
 
 %install
 make DESTDIR=%{buildroot} install
-find %{buildroot} -name '*.la' -delete
+find %{buildroot} -type f -name "*.la" -delete -print
```

Comment:
I've got nothing against it, just curious: why `-print`? Is it something the linter is adding automatically?

Reply:
We decided to unify all forms of *.la deletion, and this just happened to be what the openSUSE version of the linter suggested.